src/api/spec/controllers/trigger_controller_spec.rb

```diff
@@ -17,6 +15,12 @@
     package
   end
 
+  it 'includes Triggerable concern' do
```

Comment:
I don't think this spec brings any value. I would remove it.

Reply:
done